subsys/net/ip/net_shell.c

```diff
@@ -4632,6 +4632,285 @@ static int cmd_net_tcp(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+#if defined(CONFIG_NET_UDP) && defined(CONFIG_NET_NATIVE_UDP) && \
+		defined(CONFIG_NET_IPV6)
```

Comment:
It looks this define is a leftover. Now with IPv4 support it should be removed
```suggestion
#if defined(CONFIG_NET_UDP) && defined(CONFIG_NET_NATIVE_UDP)
```